ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/TestData.kt

```diff
@@ -0,0 +1,145 @@
+                    accent1 = PaywallColor(stringRepresentation = "#BC66FF"),
+                    accent2 = PaywallColor(stringRepresentation = "#222222"),
+                ),
+            ),
+        ),
+        localization = mapOf(
+            "en_US" to PaywallData.LocalizedConfiguration(
+                title = "Call to action for better conversion.",
+                subtitle = "Lorem ipsum is simply dummy text of the printing and typesetting industry.",
+                callToAction = "Subscribe for {{ price_per_period }}",
+                offerDetails = "{{ total_price_and_per_month }}",
+                offerDetailsWithIntroOffer = "{{ total_price_and_per_month }} after {{ sub_offer_duration }} trial",
+                offerName = "{{ sub_period }}",
+            ),
+        ),
+        assetBaseURL = Constants.assetBaseURL,
+    )
+
+    val template2Offering = Offering(
+        identifier = "Template2",
+        availablePackages = listOf(
+            Packages.weekly,
+            Packages.monthly,
+            Packages.annual,
+        ),
+        metadata = mapOf(),
+        paywall = template2,
+        serverDescription = "",
+    )
+}
+
+private object Packages {
+    val weekly = Package(
+        packageType = PackageType.WEEKLY,
+        identifier = PackageType.WEEKLY.identifier!!,
+        offering = "offering",
+        product = TestStoreProduct(
+            id = "com.revenuecat.weekly_product",
+            title = "Weekly",
+            price = Price(amountMicros = 199, currencyCode = "USA", formatted = "$1.99"),
+            description = "Weekly",
+            period = Period(value = 1, unit = Period.Unit.WEEK, iso8601 = "P1W"),
+        ),
+    )
+    val monthly = Package(
+        packageType = PackageType.MONTHLY,
+        identifier = PackageType.MONTHLY.identifier!!,
+        offering = "offering",
+        product = TestStoreProduct(
+            id = "com.revenuecat.monthly_product",
+            title = "Monthly",
+            price = Price(amountMicros = 799, currencyCode = "USA", formatted = "$7.99"),
+            description = "Monthly",
+            period = Period(value = 1, unit = Period.Unit.MONTH, iso8601 = "P1M"),
+        ),
+    )
+    val annual = Package(
+        packageType = PackageType.ANNUAL,
+        identifier = PackageType.ANNUAL.identifier!!,
+        offering = "offering",
+        product = TestStoreProduct(
+            id = "com.revenuecat.annual_product",
+            title = "Annual",
+            price = Price(amountMicros = 6799, currencyCode = "USA", formatted = "$67.99"),
+            description = "Annual",
+            period = Period(value = 12, unit = Period.Unit.MONTH, iso8601 = "P1Y"),
+        ),
+    )
+}
+
+private object Constants {
+    val images = PaywallData.Configuration.Images(
+        header = "9a17e0a7_1689854430..jpeg",
+        background = "9a17e0a7_1689854342..jpg",
+        icon = "9a17e0a7_1689854430..jpeg",
```

Comment:
Is it normal these have two dots?

Reply:
That was a bug in how the backend was storing these initially (/cc @joshdholtz 😅)
It's since been fixed, but these test images were uploaded early on.